src/components/order/StatusLabel/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCheckCircle, faClock, faDotCircle } from '@fortawesome/free-solid-svg-icons'
+
+import { OrderStatus } from 'api/operator'
+
+export type Props = { status: OrderStatus }
+
+const Wrapper = styled.div<Props>`
+  font-size: ${({ theme }): string => theme.fontSizeNormal};
```

Comment:
niiiiiiiiiiiiiiiiiiiiice :rocket:

Reply:
nice to use this theme yes, but is not overkilling how we define the "normal" size of the text :)